scripts/generate_kustomizations.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+from collections import defaultdict
+from pathlib import Path
+
+import yaml
+
+
+def _gen_kustomization_manifest(resources: list[str]):
+    return {
+        "apiVersion": "kustomize.config.k8s.io/v1beta1",
+        "kind": "Kustomization",
+        "resources": resources,
+    }
+
+
```

Comment:
I think a comment here explaining what this script is doing would be good, just for posterity.